### PR TITLE
Updated Search in header and some ornamental changes

### PR DESCRIPTION
**Pull request by Shivansh3218**

### CODE REVIEW

File: src/Components/CoinHistoryAndData/CurrencyChart.jsx [hunk 1]
```diff
@@ -1,32 +1,46 @@
...
   Area,
   XAxis,
   YAxis,
-  CartesianGrid,
   Tooltip,
 } from "recharts";
-
-import React from "react";
-import { useState, useEffect, useContext } from "react";
+import InputLabel from "@mui/material/InputLabel";
```
Comment (Souvikdeb2612) [inline src/Components/CoinHistoryAndData/CurrencyChart.jsx]: Destructure and import 10,11,12 in same line

File: src/components/CoinHistoryAndData/CoinTable.jsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        <Paper sx={{ width: "100%" }}>
+          <TableContainer component={Paper}>
+            <Table
+              sx={{ minWidth: 650 }}
+              aria-label="simple table"
+              style={{ backgroundColor: "rgb(0, 21, 41)", color: "white" }}
+            >
+              <TableHead>
+                <TableRow>
+                  <TableCell style={{ color: "white" }}>
```
Comment (Souvikdeb2612) [inline src/components/CoinHistoryAndData/CoinTable.jsx]: Table cell is repetitive  so create an array and map through it

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/CoinHistoryAndData/CurrencyChart.jsx [hunk 3]
```diff
@@ -1,32 +1,38 @@
...
+
+  const { search, setSearch } = useContext(SearchContext);
+  const [currency, setCurrency] = useState("usd");
+  const [api, setApi] = useState(
+    `https://api.coingecko.com/api/v3/coins/${search}/market_chart?vs_currency=${currency}&days=10&interval=daily`
   );
+
+  const handleCurrency = (e) => {
+    setCurrency(e.target.value);
+    console.log(currency, "coming  from charts");
```
Comment (Souvikdeb2612) [inline src/Components/CoinHistoryAndData/CurrencyChart.jsx]: remove/comment console statements

File: src/Components/CoinHistoryAndData/CurrencyChart.jsx [hunk 4]
```diff
@@ -36,46 +42,54 @@ export default function CurrencyChart() {
...
+          label="Currency"
+          style={{
+            height: "1.6rem",
+            marginLeft: "0.5rem",
+            color: "white",
+            border: "1px solid green",
+          }}
+          onChange={handleCurrency}
+        >
+          <MenuItem value={"eur"}>Euro</MenuItem>
```
Comment (Souvikdeb2612) [inline src/Components/CoinHistoryAndData/CurrencyChart.jsx]: map repetitive code

File: src/Components/CoinHistoryAndData/MainPage.jsx [hunk 5]
```diff
@@ -1,18 +1,22 @@
 import React from "react";
+
 import CoinData from "./CoinData";
 import CurrencyChart from "./CurrencyChart";
-import MenuAppBar from "./MenuAppBar";
 
 import "../CSS/MainPage.css"
```
Comment (Souvikdeb2612) [inline src/Components/CoinHistoryAndData/MainPage.jsx:8]: format imports

File: src/components/CoinHistoryAndData/CardComponent.jsx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+      <Card
+        hoverable
+        style={{
+          width: 240,
+          marginTop: "2rem",
+        }}
+        cover={
+          <img
+            alt="example"
+            src="https://media-exp1.licdn.com/dms/image/C4D03AQEMjX0D43wc5w/profile-displayphoto-shrink_800_800/0/1655134098103?e=1674086400&v=beta&t=pXjqIpbI0Jpd95Pd4CCCkQ6bnlTuzVrMv4YEEE7irik"
```
Comment (Souvikdeb2612) [inline src/components/CoinHistoryAndData/CardComponent.jsx:33]: imports static images seaparately

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/CoinHistoryAndData/CurrencyChart.jsx (hunk 1 above)
Comment (Shivansh3218) [reply src/Components/CoinHistoryAndData/CurrencyChart.jsx]: done

File: src/components/CoinHistoryAndData/CoinTable.jsx (hunk 2 above)
Comment (Shivansh3218) [reply src/components/CoinHistoryAndData/CoinTable.jsx]: done

File: src/Components/CoinHistoryAndData/CurrencyChart.jsx (hunk 3 above)
Comment (Shivansh3218) [reply src/Components/CoinHistoryAndData/CurrencyChart.jsx]: done

File: src/Components/CoinHistoryAndData/CurrencyChart.jsx (hunk 4 above)
Comment (Shivansh3218) [reply src/Components/CoinHistoryAndData/CurrencyChart.jsx]: done

File: src/Components/CoinHistoryAndData/MainPage.jsx (hunk 5 above)
Comment (Shivansh3218) [reply src/Components/CoinHistoryAndData/MainPage.jsx:8]: done

File: src/components/CoinHistoryAndData/CardComponent.jsx (hunk 6 above)
Comment (Shivansh3218) [reply src/components/CoinHistoryAndData/CardComponent.jsx:33]: didn't get it .

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**